### ISSUE TITLE
Fix validation of champva member number

### ISSUE THREAD

**Issue by michaelclement** [department-of-veterans-affairs/va.gov-team#91733]
The member number validation on the frontend does not match that on the backend. We need to fix this asap as it causes form submissions to fail.

**Comment by michaelclement** [comment]
Work done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/31647). Waiting on:
- [X] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
The above PR ensures that the front-end validation agrees with the backend validation. Whether or not we will change that validation in the future depends on policy decisions (i.e., keep the SSN formatting, use something new that matches whatever the new member structure is). 

This is meant to keep our current experience from breaking and can be revisited in the future when those decisions have been made.

**Comment by michaelclement** [comment]
| |screen|
|-|-----|
|Too short|<img width="622" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBektXQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--47db35681cc36a18d9a68d04c977b06849a73daf/Screenshot%202024-08-28%20at%2011.42.24.png" alt="Screenshot 2024-08-28 at 11.42.24.png" />|
|Numbers only|<img width="713" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeldXQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2ebfa2f221c42ed65ec207d8dd22967c763e3c07/Screenshot%202024-08-28%20at%2011.42.45.png" alt="Screenshot 2024-08-28 at 11.42.45.png" />|
| Too long | (users are not able to enter more than 9 digits) |

**Comment by marywang2** [comment]
We will need to revisit error messaging with CAIA but can do in another ticket.
